packages/memento/test/Adaptor.test.js

```diff
@@ -0,0 +1,29 @@
+import { expect } from 'chai';
+import { listLibraries } from '../src/Adaptor.js';
+import { enableMockClient } from '@openfn/language-common/util';
+
+const testServer = enableMockClient('https://api.mementodatabase.com');
```

Comment:
Can we add a test here that has pagination?

Reply:
I have added unit test for getEntries which used requestWithPagination util